src/api-reference/request/request.markdown

```diff
@@ -1,244 +1,411 @@
 ---
-title: Request
+title: Request endpoints
 layout: reference
 ---
 
-# Request
+# Request endpoints
 
-Concur Request automates the spend request and approval process for both travel and everyday expenses, giving you the data you need to accurately track and better control spending. By increasing visibility into planned expenses and up-to-date budget data, you can make strategic spending decisions before any spending actually occurs. The Request resource provides many abilities, including viewing requests and transition of requests into the workflow.
```

Comment:
The Request resources provide many abilities, including viewing Requests and transitioning Requests into the workflow.

Reply:
This was removed, no change